index.html

```diff
@@ -10,11 +10,13 @@
 
 $(document).ready(function(){
 
-    $('#button').click(function(e) {  
-        var inputvalue = $("#input").val();
-        var inputvalue2 = $("#input2").val();
-        window.location.replace(" http://www.paypal.me/"+inputvalue+"/"+inputvalue2);
+    $('#button').click(function(e) {
+      var username = $("#input").val();
+      var amount = $("#input2").val();
 
+      var token = window.btoa(username + ':' + amount).replace(/\=*$/m, '');
+
+      $('.link').text('http://exposure.money/pay/'+token);
```

Comment:
Here you can also do a `$('.copy-text').show()` where .copy-text is that "copy the link above" text